src/crux/api/ICruxAPI.java

```diff
@@ -151,4 +151,11 @@
      * @return                 a lazy sequence of the transaction log.
      */
     public Iterable<List<?>> txLog(Closeable txLogContext, Long fromTxId, boolean withDocuments);
+
+    /**
+     * Return frequencies of indexed attributes.
+     *
+     * @return         Map containing Keyword freqencies.
+     */
+    public Map<?,?> stats();
```

Comment:
Guess this can be Keyword/Integer (or Long)?
What about calling this `attributeStats` or something similar?